cime_config/acme/machines/template.case.run

```diff
@@ -1,416 +1,327 @@
+        timingfile = model + "_timing_stats." + lid
+        cmd = "gzip " + (os.path.join(caseroot, "timing", timingfile))
+        print "DEBUG: command is ",cmd
+        run_cmd(cmd) 
+
+###############################################################################
+def saveLogs(case, lid):
+###############################################################################
+
+    logdir   = case.get_value("LOGDIR")
+    if logdir is not None and len(logdir) > 0:
+        caseroot = case.get_value("CASEROOT")
+        rundir   = case.get_value("RUNDIR")
+
+        if not os.path.isdir(logdir):
+            os.makedirs(logdir)
+
+        # get components
+        files = Files()
+        config_file = files.get_value("CONFIG_DRV_FILE")
+        component = Component(config_file)
+        comps = [x.lower() for x in component.get_valid_model_components()]
+        comps = [x.replace('drv', 'cpl') for x in comps]
+        model = [case.get_value("MODEL")]
+        comps = comps + model
+
+        for comp in comps:
+            logfile = os.path.join(comp + '.log.' + lid)
+            if os.path.isfile(os.path.join(rundir,logfile)):
+                cmd = "gzip " + logfile
+                run_cmd(cmd, from_dir=rundir)
+                logfile = logfile + ".gz"
+                shutil.copy(os.path.join(rundir,logfile), 
+                            os.path.join(caseroot,logdir,logfile))
+
+###############################################################################
+def resubmitCheck(case):
+###############################################################################
+
+    # check to see if we need to do resubmission from this particular job,
+    # Note that Mira requires special logic
+
+    caseroot     = case.get_value("CASEROOT")
+    dout_s       = case.get_value("DOUT_S")
+    mach         = case.get_value("MACH")
+    testcase     = case.get_value("TESTCASE")
+    resubmit_num = int(case.get_value("RESUBMIT"))
+    
+    # If dout_s is True than short-term archiving handles the resubmit
+    # that is not the case on Mira
+    resubmit = False
+    if not dout_s and resubmit_num > 0: 
+        resubmit = True
+    elif dout_s and mach == 'mira':
+        resubmit = True
+
+    if resubmit:
+	cmd = os.path.join(caseroot,"case.submit") + "-resubmit -scriptname case."
+        if testcase is not None and testcase == 'ERR':
+	   cmd = cmd + "test"
+        else:
+	   cmd = cmd + "run"
+        logger.debug( "running resubmit check %s " %cmd)
+        rc, output, error = run_cmd(cmd, ok_to_fail=True, from_dir=caseroot)
+        if rc != 0:
+            logger.warn("could not run %s submit script, rc=%s, output=%s, error=%s" %(model,rc,output,error))
+
+###############################################################################
+def DoDataAssimilation(case, da_script, lid):
+###############################################################################
+
+    cmd = "da_script 1> da.log.%s 2>&1" %(lid)
```

Comment:
We should definitely have some kind of data assimilation test - since the regular testing did not pick this up.